cli/src/main.rs

```diff
@@ -105,42 +101,24 @@ pub fn parse_args(matches: &ArgMatches<'_>) -> Result<CliConfig, Box<dyn error::
     } = parse_command(&matches)?;
 
     let (keypair, keypair_path) = if require_keypair {
-        let KeypairWithSource { keypair, source } = keypair_input(&matches, "keypair")?;
```

Comment:
Looks like you can remove the global `ASK_SEED_PHRASE_ARG` arg now

Reply:
Nice catch!

Reply:
Ugh... Github, what are you doing?